tests/test_wave_05.py

```diff
@@ -123,28 +133,29 @@ def test_delete_goal(client, one_goal):
     # Check that the goal was deleted
     response = client.get("/goals/1")
     assert response.status_code == 404
+    assert Goal.query.get(1) == None
```

Comment:
Besides just testing the result of deleting a record from the database, we should also check that the response that from the GET request to a goal that was deleted is what you expect it to be. 

What would your assertion for checking the response body look like?

Reply:
✅ Added this!